Automattic/abacus
# Experiment: add route flexibility

This is a potential solution for https://github.com/Automattic/experimentation-platform/issues/957 to prevent an app crash when the URL param doesn't exactly match the experiment `/:view`, by rendering the default view and logging the validation error to the console.

## How has this been tested?






- Manual testing with production data (locally)
- Additional manual testing instructions (please specify):
  * add `/bad-param` to an experiment URL and the default view should be loaded (eg: for status `running` => `/results` view).
